android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -1,3 +1,10 @@
+var _debug = 1;
```

Comment:
Do we want to check this in as `0`?

Reply:
*snap* -- see my comment at exact same time!